x-pack/plugins/enterprise_search/public/applications/app_search/components/settings/log_retention/log_retention_confirmation_modal.tsx

```diff
@@ -20,7 +20,7 @@ export const LogRetentionConfirmationModal: React.FC = () => {
   const CANNOT_BE_RECOVERED_TEXT = i18n.translate(
     'xpack.enterpriseSearch.appSearch.settings.logRetention.modal.recovery',
     {
-      defaultMessage: 'Once your data has been removed, it cannot be recovered.',
+      defaultMessage: 'You can not recover deleted data.',
```

Comment:
It kind of surprises me this is `can not` vs `cannot` 🤔 @gchaps, was this intentional?

Reply:
Good catch. Should be cannot.